www/js/menu.js

```diff
@@ -301,6 +304,68 @@ const announcementSlideButton = h(
   ),
 );
 
+const createSyncModal = (content, code) => {
+  const syncModal = new tingle.Modal({
+    footer: true,
+    stickyFooter: false,
+    cssClass: ['sync-code-modal'],
+    closeMethods: ['overlay', 'button', 'escape'],
+  });
+  // open modal
+  syncModal.open();
+
+  if (code) {
+    syncModal.addFooterBtn('Copy', 'tingle-btn tingle-btn--primary', () => {
+      copy(code);
+      syncModal.close();
+      syncModal.destroy();
+    });
+    syncModal.addFooterBtn('End Session', 'tingle-btn tingle-btn--primary', () => {
+      onEnd(code);
+      syncModal.close();
+      syncModal.destroy();
+      document.getElementById('remote-sync-icon').style.color = '#424242';
```

Comment:
I think it would be better if we add a class here, and then manipulate that through css. What do you think?

It just gets a bit hard managing styles when they are inside JS. Especially if I want to add more css rules here.

Reply:
this is temporary based on the mocks @saintsoldierx had. I can change it if you'd like